src/router/kromo/dd-wrt/elegant/fresh-dark.css

```diff
@@ -162,13 +162,13 @@ table tr:not([bgcolor]):hover {
   border-color: #333;
 }
 td[bgcolor="#C0C0C0"] {
-	background-color: #383838;
+  background-color: #383838;
 }
 td[bgcolor="#C0C0C0"]:hover {
-	background-color: #1a1a1a;
+  background-color: #1a1a1a;
 }
 form[name="wds"] select {
-	margin-right: 6px;
+  margin-right: 6px;
 }
```

Comment:
indent fix only on these 3 rules  to conform to the indenting style, this is consistency only and not mixing tabs/spaces.